File: SisControl.NET/Formularios/Producao/frmRepicadores.vb
Public Class frmRepicadores

    Private Sub RepicadorBindingNavigatorSaveItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles RepicadorBindingNavigatorSaveItem.Click
        Me.Validate()
        Me.RepicadorBindingSource.EndEdit()
        Me.RepicadorTableAdapter.Update(Me.DsCadastro.repicador)

    End Sub

    Private Sub frmRepicadores_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        'TODO: This line of code loads data into the 'DsCadastro.repicador' table. You can move, or remove it, as needed.
        Me.RepicadorTableAdapter.Fill(Me.DsCadastro.repicador)

        'Carrega o combo com a lista dos Repicadores
        Carrega_Lista(cmbRepicador, "Repicador", "id", "Nome", True)
    End Sub

    Private Sub cmbRepicador_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cmbRepicador.SelectedIndexChanged
        'move o registro atual para coincidir com a Combobox
        RepicadorBindingSource.Position = RepicadorBindingSource.Find("id", cmbVal(cmbRepicador))
    End Sub
End Class

File: SisControl.NET/Formularios/Producao/frmRepicadores.cs
public partial class frmRepicadores
{
    private void RepicadorBindingNavigatorSaveItem_Click(System.Object sender, System.EventArgs e)
    {
        this.Validate();
        this.RepicadorBindingSource.EndEdit();
        this.RepicadorTableAdapter.Update(this.DsCadastro.repicador);
    }

    private void frmRepicadores_Load(System.Object sender, System.EventArgs e)
    {
        // TODO: This line of code loads data into the 'DsCadastro.repicador' table. You can move, or remove it, as needed.
        this.RepicadorTableAdapter.Fill(this.DsCadastro.repicador);

        // Carrega o combo com a lista dos Repicadores
        Biblioteca.Carrega_Lista(ref cmbRepicador, "Repicador", "id", "Nome", true);
    }

    private void cmbRepicador_SelectedIndexChanged(System.Object sender, System.EventArgs e)
    {
        // move o registro atual para coincidir com a Combobox
        RepicadorBindingSource.Position = RepicadorBindingSource.Find("id", Biblioteca.cmbVal(ref cmbRepicador));
    }
}
